docs/references/helm-operator-integration.md

```diff
@@ -125,7 +125,7 @@ spec:
         tag: version
 ```
 
-#### Filter
+#### Filters (deprecated)
```

Comment:
Filters are not deprecated, the preferred annotation for it just changed from `fluxcd.io/tag.<alias>` to `filter.fluxcd.io/<alias>`.

Reply:
Does `chart-image` still have the same effect if supplied as the alias for `filter.fluxcd.io/<alias>`?

Reply:
Yes, `chart-image` is a static alias for an auto detected top-level image but functions like any other alias.

Reply:
Ok, thanks @hiddeco. Updated :+1: